### ISSUE TITLE
Should parsing mention ARIA and the accessibility API?

### ISSUE THREAD

**Issue by chaals** [w3c/html#106]
This was https://www.w3.org/Bugs/Public/show_bug.cgi?id=13511

I **think** we should clarify that we expect an accessibility tree to be built, when the parser comes across aria stuff.

I don't know the parsing section well enough to propose a fix yet, but I'm hoping it's reasonably straightforward to put the pointer in the right place, and hand off to the ARIA mappings specs.

**Comment by travisleithead** [comment]
@cyns to weigh in on this one.

**Comment by travisleithead** [comment]
Lumping in Bug https://www.w3.org/Bugs/Public/show_bug.cgi?id=24571 as well.

> same issue as 13511, but for 5.1.  Need to cover in aapi mapping doc, and link from main html 5.1 spec.

**Comment by stevefaulkner** [comment]
this needs someone with undertsanding of parser to contribute

**Comment by stevefaulkner** [comment]
reading back through the original bug cynthia noted that this should be covered in the acc API docs. will close here and open on HTML - API spec
